pkg/products/cloudresources/reconciler.go

```diff
@@ -178,6 +177,15 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, nil
 	}
 
+	if r.Config.GetStrategiesConfigMapName() == "" {
+		err := r.setPlatformStrategyName(ctx, client)
```

Comment:
The primary reason was that this now needs a k8sclient to determine the platform and none of the existing calls to `NewReconciler` use one. This could be refactored though to instantiate the k8sclient earlier in rhmi_controller and pass it in to be used by the cloud resources reconciler.

Reply:
ok, i'm happy for that to be addressed at a later time, if we can create a follow on JIRA for it. 👍

Reply:
[MGDAPI-5502](https://issues.redhat.com/browse/MGDAPI-5502)